### ISSUE TITLE
Separate Buildable and Passable object tags

### ISSUE THREAD

**Issue by Julian-B** [baxter-oop/AndOre#119]
Just that. Rewrite objects so that players can pass through an object, but not build upon it. And vice-versa.

**Comment by JCharante** [comment]
Yeah just add seperate flags for being passable and being build-upon-able
